tunnel/tunnel.go

```diff
@@ -201,11 +201,11 @@ type Tunnel struct {
 }
 
 // New creates a new instance of Tunnel.
-func New(tunnelType string, server *Server, source, destination []string) (*Tunnel, error) {
+func New(tunnelType string, server *Server, source, destination []string, config string) (*Tunnel, error) {
```

Comment:
You don't need the `config` variable defined here since you can use the value kept on `server`.

Reply:
We have no direct config value in a Server, ain't we? Especially the LocalForward and RemoteForward values we need.
![image](https://user-images.githubusercontent.com/40490784/95518470-c307a000-09cb-11eb-8379-99814fc24b95.png)